cypress/e2e/court-cases/court-case-details/TriggersAndExceptions.cy.ts

```diff
@@ -378,7 +378,7 @@ describe("Triggers and exceptions tabs", () => {
   })
 
   it("should only show triggers to Trigger Handlers", () => {
-    newUserLogin({ groups: [UserGroup.TriggerHandler] })
+    newUserLogin({ groups: [UserGroup.TriggerHandler], hasAccessToTriggers: true })
```

Comment:
I'm happy to approve this but I wonder if it's something that maybe we'd want to infer from `newUserLogin` from the groups passed in 🤔

Reply:
I had a niggling feeling when we removed the shared implementations of those functions and this might be why

Reply:
that's a really good point, I'll update put the util functions back in and use those

Reply:
Changed in https://github.com/ministryofjustice/bichard7-next-ui/pull/326/commits/b1c9a5803e80c5b5f1953114b1084e38424f6a9e